terraform/environments/cooker/macie.tf

```diff
@@ -0,0 +1,32 @@
+
+# Create macie account
+# When logging in as adminisitrator this is not needed but may be for our customers
+# Account name is MacieAccount. Publishes data every hour (choice are FIFTEEN_MINUTES, ONE_HOUR or SIX_HOURS)
+
+resource "aws_macie2_account" "macieaccess" { 
+   finding_publishing_frequency = "ONE_HOUR"
+   status                       = "ENABLED"
+ }
+
+# Now create a job
+# Job type options are ONE-TIME and SCHEDULED. If scheduled is chosen you need to select a frequency using schedule_frequency of daily_schedule, weekly_schedule or monthly_schedule
+# The buckets should be replaced but a list of buckets to examine. This could be limited to those in the account you are connected to or others to which you have access. 
+# Below there is a limited list of those from example shown. 
+# There are many other job options available and these should be looked at in terrform [job creation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/macie2_classification_job)
+
+resource "aws_macie2_classification_job" "test" {
+  job_type = "ONE_TIME"
+  name     = "JOBNAME"
+  s3_job_definition {
+    bucket_definitions {
+      account_id = "236861075084"
```

Comment:
Account ID should be retrieved programmatically, rather than being hardcoded. There are plenty of examples in this repository of how to do this.

Reply:
Amended this.